core/src/main/java/org/infinispan/interceptors/ClusteredCacheLoaderInterceptor.java

```diff
@@ -41,24 +41,35 @@ private void startClusteredCacheLoaderInterceptor() {
    }
 
    @Override
-   protected boolean skipLoadForWriteCommand(WriteCommand cmd, Object key, InvocationContext ctx) {
+   protected boolean skipLoadForWriteCommand(WriteCommand cmd, Object key, InvocationContext ctx, boolean functionalReadWrite) {
```

Comment:
Right. I was not sure about that either. Earlier versions (still immortalized by earlier commits in this PR) used instanceof checks deeper in the call stack but I was even less sure about those :) Would these checks be more acceptable or should I go the functional-specific methods way?